### PR TITLE
Remove terraform-modules-public package

### PR DESCRIPTION
**Pull request by jcortejoso**
### Description

Remove terraform-modules-public as they were discontinued and not maintained. Replaced by https://github.com/celo-org/charts/tree/main/charts.

https://github.com/celo-org/infrastructure/issues/1105

### CODE REVIEW

File: packages/docs/developer-resources/utils/reference/README.md
Comment (lvpeschke) [inline packages/docs/developer-resources/utils/reference/README.md:1]: It seems to me the canonical file is at https://github.com/celo-org/celo-monorepo/blob/master/README.md.
I would remove this one entirely and instead update the other README.

File: packages/terraform-modules-public/images/readme.md
Comment (lvpeschke) [inline packages/terraform-modules-public/images/readme.md:1]: Are these images cleaned up already?

**Review by lvpeschke — APPROVED** [review]

File: packages/docs/developer-resources/utils/reference/README.md
Comment (jcortejoso) [reply packages/docs/developer-resources/utils/reference/README.md:1]: Thanks for the note. It seems that this file is generated from `phone-utils` package with the command `yarn run typedoc`. I've updated it.

File: packages/terraform-modules-public/images/readme.md
Comment (jcortejoso) [reply packages/terraform-modules-public/images/readme.md:1]: I've deleted the old ones in GCP. We had an updated version from Dec. 9th, 2022. I think we can keep those for now.

**Pull request merged**